scripts/rnnlm/compute_sentence_scores.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Copyright 2017  Hainan Xu
+#           2017  Szu-Jui Chen
+
```

Comment:
it looks to me like your changes to this script would break the old use-case.  I think it would make more sense to have a separate version of this script to do the bidirectional rescoring, and you should talk to @hainan-xv about this.  Personally I think it's a mistake to add this kind of thing in the Chime recipe as it's a very separate issue from the focus of the chime challenge, and it just makes the baseline more of a hassle to run.